master/getting-started/docker/installation/manual.md

```diff
@@ -1,101 +1,30 @@
 ---
-title:  Preparing the environment for Calico as a Docker network plugin
+title: Installing Calico for Docker
 ---
 
-The worked example in the _Calico as a Docker network plugin tutorial_ is run
-on two Linux servers that have a number of installation requirements.
+On each host, perform the following:
 
-This tutorial describes how to manually configure a working environment for
-the example.
+1. Download the calicoctl binary:
 
-## Requirements
+	 ```
+   sudo wget -O /usr/local/bin/calicoctl http://www.projectcalico.org/builds/calicoctl
+	 sudo chmod +x calicoctl
+   ```
 
-You will need 2 servers (bare metal or VMs) with a  modern 64-bit Linux OS
-and IP connectivity between them.
+2. Launch `calico/node`:
 
-We recommend configuring the hosts with the hostname `calico-01` and
-`calico-02`.  The worked example will refer to these hostnames.
+   ```
+   sudo calicoctl node run
+   ```
 
-They must have the following software installed:
 
-- Docker 1.9 or greater (details below)
-- `ip_set`, `iptables`, and `ip6tables` kernel modules.
--  The `calicoctl` binary in your path (see below)
+Check that your installation was successful with the following command:
 
-You will also need an etcd cluster which Calico uses for coordinating state
-between the nodes.  This may installed on one or both of the two servers for
-the worked example.  See the [etcd documentation][etcd] for details on setting
-up a cluster.
+```
+calicoctl node checksystem
+```
```

Comment:
I think it is worth stating that the node run command is a shortcut for launching the calico/node container image.   If you want to run the daemon in an init system, use the --init-system option to display the relevant start and stop commands to use in your init scripts (or there might be a different doc to point at).

I'd be happy with an issue being raised for this point so that it doesn't delay you getting it  in.